.changeset/calm-moons-run.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/number-input': minor
+---
+
+prevent number input change when scrolling with the mouse.
```

Comment:
```suggestion
Preventing the value of the input to be updated when it has the focus and the user scrolls the page content using the mouse wheel.
```

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2750/commits/5020ad798c4a22d7ab62fb41ec3a68391b43f509`

Thank you.